### PR TITLE
ARMChecker bug fix, OrgPolicy for Azure Gov and Azure policy changes

### PR DESCRIPTION
**Pull request by Akreddy123**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Core/ARMChecker/ARMCheckerStatus.ps1 [hunk 1]
```diff
@@ -431,16 +447,24 @@ class ARMCheckerStatus: EventBase
...
+	} 
 	
 	hidden [string] LoadARMControlsFile()
 	{ 	
 	   $serverFileContent=$null;
 	   $ARMControlsFileURI = [Constants]::ARMControlsFileURI
 	   try
 	   {
-	    $serverFileContent = [ConfigurationHelper]::InvokeControlsAPI($ARMControlsFileURI, '', '', '');
+         $AzureContext = Get-AzureRmContext
```
[the remark this answers is not in the document]
Comment (Akreddy123) [reply src/AzSK/Framework/Core/ARMChecker/ARMCheckerStatus.ps1]: we don't set AzurermContext in ArmChecker, so Helpers]::currentRMContext  will not be intialized

File: src/AzSK/PolicySetup/PolicySetup.ps1 [hunk 2]
```diff
@@ -59,6 +59,11 @@ function Install-AzSKOrganizationPolicy
 		[Alias("ainame")]
 		$AppInsightName,
 
+		[Parameter(Mandatory = $false, HelpMessage = "Provide your Azure Environment")]
+        [string]
+		[Alias("ae")]
+        $AzureEnvironment = "AzureCloud",
```
Comment (VishalHaibatpure) [inline src/AzSK/PolicySetup/PolicySetup.ps1]: use same variable in Update-Org command as well.

File: src/AzSK/Framework/Core/ARMChecker/ARMCheckerStatus.ps1 [hunk 3]
```diff
@@ -431,16 +447,24 @@ class ARMCheckerStatus: EventBase
...
+	  if([ConfigurationManager]::GetLocalAzSKSettings().EnableAADAuthForOnlinePolicyStore -and -not [string]::IsNullOrWhiteSpace($AzureContext)) 
+	   {
+		   $serverFileContent = [ConfigurationManager]::LoadServerConfigFile("ARMControls.json");
 	   }
+	   else
+	   {
+		   $serverFileContent = [ConfigurationHelper]::InvokeControlsAPI($ARMControlsFileURI, '', '', '');
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/ARMChecker/ARMCheckerStatus.ps1]: you can use [WebRequestHelper]::InvokeGetWebRequest

File: src/AzSK/Framework/Core/PolicySetup/PolicySetup.ps1 [hunk 4]
```diff
@@ -411,8 +414,9 @@ class PolicySetup: CommandBase
 			$this.ValidatePolicyExists()
 		}
 
-		
+		if($this.AzureEnvironment -eq "AzureCloud"){
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/PolicySetup/PolicySetup.ps1]: Use enum instead of Strings

**Pull request merged**